pkg/virt-api/api.go

```diff
@@ -780,7 +780,7 @@ func (app *virtAPIApp) Compose() {
 }
 
 func (app *virtAPIApp) ConfigureOpenAPIService() {
-	config := openapi.CreateConfig()
+	config := openapi.CreateV3Config()
```

Comment:
Why is this needed?

Reply:
At the time it looks like it was necessary but on second look I think it can be removed. DONE